### PR TITLE
[WIP] Docs cleanup

### PR DESCRIPTION
**Pull request by zeke**
Just making my way through the documentation, cleaning up typos and stuff.

📚

### CODE REVIEW

File: docs/faq/electron-faq.md [hunk 1]
```diff
@@ -78,8 +77,8 @@ app.on('ready', function() {
 ## I can not use jQuery/RequireJS/Meteor/AngularJS in Electron.
 
 Due to the Node.js integration of Electron, there are some extra symbols
-inserted into DOM, like `module`, `exports`, `require`. This causes troubles for
-some libraries since they want to insert the symbols with same names.
+inserted into the DOM like `module`, `exports`, `require`. This causes troubles for
```
Comment (enlight) [inline docs/faq/electron-faq.md]: While you're at it, `troubles` -> `problems`.

Comment (zeke) [inline docs/faq/electron-faq.md]: Done. thanks, @enlight.

**Comment by kevinsawicki** [comment]
This looks great to me, feel free to merge :ship: :sheep: :shipit:

**Comment by zcbenz** [comment]
:+1:

**Comment by zeke** [comment]
Thanks! Will pick up further cleanup efforts in a new PR.

**Pull request merged**